vllm/platforms/cuda.py

```diff
@@ -141,6 +141,13 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         if cache_config and cache_config.block_size is None:
             cache_config.block_size = 16
 
+    @classmethod
+    def get_current_memory_usage(cls,
```

Comment:
Please add the same one for ROCm, which is also a cuda-alike platform.

Reply:
Done.